### PR TITLE
Update global pipelines cypress test

### PR DESCRIPTION
**Pull request by ppadti**
Closes: [RHOAIENG-3820](https://issues.redhat.com/browse/RHOAIENG-3820)

## Description
This PR aims to add additional cypress test to global pipeline page. That includes:
- Configure pipeline server
-  delete pipeline server
- view pipeline configuration
- table sorting, filtering.

## How Has This Been Tested?
`npm run test`

## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

File: frontend/src/__tests__/cypress/cypress/e2e/pipelines/Pipelines.cy.ts [hunk 1]
```diff
@@ -32,17 +38,259 @@ const initialMockPipelineVersion = buildMockPipelineVersionV2({
...
+        namespace: projectName,
+      }),
+    ).as('createDSPA');
+
+    pipelinesGlobal.findConfigurePipelineServerButton().should('be.enabled');
+    pipelinesGlobal.findConfigurePipelineServerButton().click();
+    configurePipelineServerModal.findAwsKeyInput().type('test-aws-key');
+    configurePipelineServerModal.findAwsSecretKeyInput().type('test-secret-key');
+    configurePipelineServerModal.findEndpointInput().type('https://s3.amazonaws.com/');
+    configurePipelineServerModal.findRegionInput().type('us-east-1');
```
Comment (pnaik1) [inline frontend/src/__tests__/cypress/cypress/e2e/pipelines/Pipelines.cy.ts:74]: The region field has default value `us-east-1`

File: frontend/src/__tests__/cypress/cypress/e2e/pipelines/Pipelines.cy.ts [hunk 2]
```diff
@@ -32,17 +38,259 @@ const initialMockPipelineVersion = buildMockPipelineVersionV2({
...
+                accessKey: 'AWS_ACCESS_KEY_ID',
+                secretKey: 'AWS_SECRET_ACCESS_KEY',
+                secretName: 'test-secret',
+              },
+            },
+          },
+        },
+      });
+    });
+  });
```
Comment (pnaik1) [inline frontend/src/__tests__/cypress/cypress/e2e/pipelines/Pipelines.cy.ts:118]: Can you please add test cases for prefill from a Data Connection dropdown?

File: frontend/src/__tests__/cypress/cypress/e2e/pipelines/Pipelines.cy.ts [hunk 3]
```diff
@@ -32,17 +38,259 @@ const initialMockPipelineVersion = buildMockPipelineVersionV2({
...
+
+    pipelinesGlobal.findConfigurePipelineServerButton().should('be.enabled');
+    pipelinesGlobal.findConfigurePipelineServerButton().click();
+    configurePipelineServerModal.findAwsKeyInput().type('test-aws-key');
+    configurePipelineServerModal.findAwsSecretKeyInput().type('test-secret-key');
+    configurePipelineServerModal.findEndpointInput().type('https://s3.amazonaws.com/');
+    configurePipelineServerModal.findRegionInput().type('us-east-1');
+    configurePipelineServerModal.findBucketInput().type('test-bucket');
+    configurePipelineServerModal.findConfigurePipelineButton().should('be.enabled');
+    configurePipelineServerModal.findConfigurePipelineButton().click();
```
Comment (pnaik1) [inline frontend/src/__tests__/cypress/cypress/e2e/pipelines/Pipelines.cy.ts:77]: Can you add please test cases for database input as well?

**Review by pnaik1 — CHANGES_REQUESTED** [review]

**Review by alexcreasy — APPROVED** [review]
Looks good!
/approve

**Pull request merged**